specification/maps/data-plane/Search/preview/2022-12-01-preview/search.json

```diff
@@ -539,7 +539,7 @@
     },
     "/reverseGeocode": {
       "get": {
-        "description": "**Reverse Geocode a point.**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nTranslate a coordinate (example: 37.786505, -122.3862) into a human understandable street address. Most often this is needed in tracking applications where you receive a GPS feed from the device or asset and wish to know what address where the coordinate is located. This endpoint will return address information for a given coordinate.\n\n>[!Important]\n>By using this feature, you agree to the preview legal terms. See the [Preview Supplemental Terms](https://azure.microsoft.com/en-us/support/legal/preview-supplemental-terms/) for additional details.\n\n",
+        "description": "**Reverse Geocode a point.**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nTranslate a coordinate (example: 37.786505, -122.3862) into a human understandable street address. Most often this is needed in tracking applications where you receive a GPS feed from the device or asset and wish to know what address where the coordinate is located. This endpoint will return address information for a given coordinate.\n\n>[!Important]\n>This feature is not yet available, and this document describes a preview of the specification. The API will respond with HTTP status code 404 Not Found. \nBy using this feature, you agree to the preview legal terms. See the [Preview Supplemental Terms](https://azure.microsoft.com/en-us/support/legal/preview-supplemental-terms/) for additional details.\n\n",
```

Comment:
The expectation is that the Swagger describes the service. It makes no sense to publish a path here if the service does not actually support it. If this is not supported by the service, this entire path should be removed.

Reply:
@tjprescott the swagger does describe the service. The service will support this exact design as it's been approved by the board. If we remove the paths, then will you approve the breaking change once the service is enabled? Otherwise, can you provide us a better solution or escalation path for visibility? Currently this spec is [already published](https://learn.microsoft.com/en-us/rest/api/maps/search-v2)

Reply:
I don't understand when you say it "will" support the design. The preview version of the service either supports the endpoint or it doesn't (and it sounds like it doesn't). If a later preview adds support for the endpoint, then that is when the endpoint should appear in Swagger.

I would approve the breaking change in this PR to remove the endpoint if it was added erroneously.

Reply:
Agreed, the issue is the services are not developed at the same time but the specs we're reviewed at the same time and respectively checked in and published. The issue to avoid is going through another round of spec review if we remove the specs from this PR. We'll link back to PR/commit history to show it was approved though.

@john35452 can you please remove the revgc operations per request here and immediately tag @tjprescott for review so that we can catch the docs publishing train asap.